src/redux/modules/fullConceptsCompare.js

```diff
@@ -0,0 +1,123 @@
+import _ from 'lodash';
+import { reduxFsaThunk } from '../../lib/redux-fsa-thunk';
+import { resourceServiceApiGet } from '../../api/resource-service-api/host';
+
+export const FULL_CONCEPTSCOMPARE_REQUEST = 'FULL_CONCEPTSCOMPARE_REQUEST';
+export const FULL_CONCEPTSCOMPARE_SUCCESS = 'FULL_CONCEPTSCOMPARE_SUCCESS';
+export const FULL_CONCEPTSCOMPARE_FAILURE = 'FULL_CONCEPTSCOMPARE_FAILURE';
+
+export const ADD_FULL_CONCEPT_TO_COMPARE = 'ADD_FULL_CONCEPT_TO_COMPARE';
+export const REMOVE_FULL_CONCEPT_TO_COMPARE = 'REMOVE_FULL_CONCEPT_TO_COMPARE';
+
+function shouldFetch(metaState) {
+  const threshold = 60 * 1000; // seconds
+  return (
+    !metaState ||
+    (!metaState.isFetching &&
+      (metaState.lastFetch || 0) < Date.now() - threshold)
+  );
+}
+
+export function fetchFullConceptsToCompareIfNeededAction() {
+  return (dispatch, getState) => {
+    const ids = Object.keys(getState().conceptsCompare.items);
+    ids
+      .filter(id => !!id)
+      .forEach(id => {
```

Comment:
Du kan sende en POST til /concepts med body `{"ids": [list-med-alle-idene]}`, i stedet for en GET per id